psalm.xml

```diff
@@ -14,6 +14,13 @@
         </ignoreFiles>
     </projectFiles>
 
+    <issueHandlers>
+        <DeprecatedClass>
+            <errorLevel type="suppress">
+                <referencedClass name="Laminas\ServiceManager\Config"/>
+            </errorLevel>
+        </DeprecatedClass>
+    </issueHandlers>
```

Comment:
Please remove this suppression here. Usage of this deprecated class can be fixed by just passing configuration arrays to the `ServiceManager` constructor in tests where it is used:

- SessionConfigFactoryTest
- ContainerAbstractServiceFactoryTest
- SessionManagerFactoryTest
- StorageFactoryTest

Reply:
I tried this but not yet resolved few more failures are introduce Can you please check?

Reply:
> Please remove this suppression here. Usage of this deprecated class can be fixed by just passing configuration arrays to the `ServiceManager` constructor in tests where it is used:
> 
> * SessionConfigFactoryTest
> * ContainerAbstractServiceFactoryTest
> * SessionManagerFactoryTest
> * StorageFactoryTest

It may be something wrong with my fix Can you please check and suggest lot of unit tests and same deprecations psalm errors are also coming?

Reply:
done